src/AcceptanceTests.SqlServer/MultiSchema/When_custom_schema_configured_for_endpoint.cs

```diff
@@ -0,0 +1,102 @@
+﻿namespace AcceptanceTests.SqlServer.MultiSchema
+{
+    using System.Threading.Tasks;
+    using NServiceBus;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.Features;
+    using NServiceBus.Transport.SqlServer;
+    using NUnit.Framework;
+    using Conventions = NServiceBus.AcceptanceTesting.Customization.Conventions;
+
+    public class When_custom_schema_configured_for_endpoint : BridgeAcceptanceTest
+    {
+        public const string PublisherSchema = "publisher";
+
+        [Test]
+        public async Task Subscriber_should_get_the_event()
+        {
+            var context = await Scenario.Define<Context>()
+            .WithBridge(bridgeConfiguration =>
+            {
+                var endpointAddress = GetTestEndpointAddress<Publisher>();
+                var bridgeTransport = new BridgeTransport(TransportBeingTested);
+                var endpointName = Conventions.EndpointNamingConvention(typeof(Publisher));
+                bridgeTransport.AddTestEndpoint<Publisher>();
+                //bridgeTransport.HasEndpoint(endpointName, "WhenCustomSchemaConfiguredForEndpoint.Publisher@[publisher]@[NServiceBus]");
```

Comment:
If I hardcode the endpoint with the schema name the test will pass, but if I try to add the test endpoint with the `Publisher` class the schema is defaulted to [dbo] instead of [publisher]